src/components/Header.js

```diff
@@ -17,9 +18,11 @@ const Header = ({ pageContext, toggleBackdrop, ...props }) => {
   `)
   return (
     <header id="masthead" className="">
-      <Navbar expand="lg">
+      <Navbar expand="lg" bg="dark" variant="dark">
         <Container>
-          <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
+          <Navbar.Brand href="#home">
+            <img src={logo} alt="" />
```

Comment:
This needs to be a Gatsby Image. [StaticImage](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/#using-the-gatsby-image-components) because its located in the project.

Reply:
Image has been updated to Static Image.